icons/ndst.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```

Comment:
I like the icon and it will be a good addition for this extension. As the colors for all icons must be picked out of the Material Color palette it would be great if you could replace the color `#3790b0` with another color which can be found via the [Material Color Converter](https://pkief.github.io/material-color-converter/).

Reply:
Of course, i change it for #0097a7.
Thanks you for the feedback.